tests/python/accuracy/public_scope.json

```diff
@@ -6,7 +6,55 @@
             {
                 "image": "coco128/images/train2017/000000000074.jpg",
                 "reference": [
-                    "[  1   2   6  11  12 126 127]"
+                    "1 2 6 11 12 126 127 "
+                ]
+            }
+        ]
+    },
+    {
+        "name": "otx_models/Lite-hrnet-18.xml",
+        "type": "SegmentationModel",
+        "test_data": [
+            {
+                "image": "coco128/images/train2017/000000000074.jpg",
+                "reference": [
+                    "0 1 "
```

Comment:
I agree that this is not enough and should be enhanced. Normally, we should take references from OTX, and save them here. Maybe @sovrasov can help us to get such references. It should not take a lot of time. It's just a processed output 
 for one sample.

Reply:
As soon as MAPI is a standalone tool, I'd propose not to tie to references from OTX. Instead, we may check C++ vs python output. Inconsistencies with OTX could be fixed once during the integration and later validation pytorch task vs openvino task will keep the OV wrappers in shape.